weed/shell/shell_liner.go

```diff
@@ -116,13 +126,14 @@ func processEachCmd(reg *regexp.Regexp, cmd string, commandEnv *CommandEnv) bool
 			for _, c := range Commands {
 				if c.Name() == cmd || c.Name() == "fs."+cmd {
 					if err := c.Do(args, commandEnv, os.Stdout); err != nil {
-						fmt.Fprintf(os.Stderr, "error: %v\n", err)
+						glog.V(0).Infof("error: %v", err)
```

Comment:
these 2 lines are the same?

Reply:
glog & fmt, glog goes into stderr and fmt does stdout.

Only the fmt line goes into file when piped as output ` > out.txt`

Maybe there's a better way to do it but I didn't see glog & stdout. just stderr and reference to why people have made other logger libraries.